src/OrleansProviders/Streams/Memory/MemoryStreamQueueGrain.cs

```diff
@@ -1,57 +1,37 @@
-﻿using System;
+﻿
+using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
 
-namespace Orleans.Providers.Streams.Memory
+namespace Orleans.Providers
 {
     /// <summary>
     /// Memory stream queue grain. This grain works as a storage queue of event data. Enqueue and Dequeue operations are supported.
     /// the max event count sets the max storage limit to the queue.
     /// </summary>
     public class MemoryStreamQueueGrain : Grain, IMemoryStreamQueueGrain
     {
-        /// <summary>
-        /// Type name. Default to the grain name.
-        /// </summary>
-        public const String TypeName = "MemoryStreamQueueGrain";
-
-        /// <summary>
-        /// Event queue. 
-        /// </summary>
-        private Queue<MemoryEventData> eventQueue = new Queue<MemoryEventData>();
+        private readonly Queue<MemoryMessageData> eventQueue = new Queue<MemoryMessageData>();
```

Comment:
I remember adding comments based solely on the warnings I got on VS, so I think we might have to do that.